nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -432,7 +432,7 @@ def create_eval_dataset(self) -> TDataset:
     def _get_pixel_sampler(self, dataset: TDataset, *args: Any, **kwargs: Any) -> PixelSampler:
         """Infer pixel sampler to use."""
         if self.config.patch_size > 1:
-            return PatchPixelSampler(*args, **kwargs, patch_size=self.config.patch_size)
+            return PairPixelSampler(*args, **kwargs, radius=self.config.patch_size)
```

Comment:
I don't think we want to make the assumption that the user wants the Pair sampler over the Patch sampler.

Reply:
Yeah I agree, do you have any suggestions of the best way to switch between the two samplers. It could be a flag or another class defined in the config file.

Reply:
I think one option is just defining a new SamplingConfig which is future proof to all the random sampling changes we might want (eg this would make it easier to implement sampling wrt image frequency, edges, etc)

Reply:
this might involve refactoring the way get_pixel_sampler works in favor of making a tyro config

Reply:
Added `PixelSamplerConfig` to define which pixel sampler to use in the datamanager, removed `EquirectangularPixelSampler` and moved its sample method to `PixelSampler` and added a check that if all images are equirectangular then use equirectangular sampling.